tests/linearizability/traffic.go

```diff
@@ -18,17 +18,21 @@ import (
 	"context"
 	"fmt"
 	"math/rand"
+	"strings"
 	"time"
 
+	"go.uber.org/zap"
 	"golang.org/x/time/rate"
 )
 
 var (
 	DefaultTraffic Traffic = readWriteSingleKey{key: "key", writes: []opChance{{operation: Put, chance: 100}}}
+	AuthTraffic    Traffic = authTraffic{readWriteSingleKey{key: "key", writes: []opChance{{operation: Put, chance: 100}}}}
 )
 
 type Traffic interface {
-	Run(ctx context.Context, c *recordingClient, limiter *rate.Limiter)
+	Run(ctx context.Context, c *recordingClient, limiter *rate.Limiter, lg *zap.Logger)
+	AuthEnabled() bool
```

Comment:
I think traffic can handle it's own setup. Maybe instead of exposing `AuthEnabled() bool` we have `Setup(ctx context.Context, c interfaces.Client, lg *zap.Logger)`

Reply:
Okay. I can add one auth related configuration to handle the set up.